ui/src/components/layout/header.js

```diff
@@ -73,20 +90,33 @@ class Header extends React.Component {
                     />*/}
                     </div>
                     <div>
-                        <div className="user" onClick={this.toggleDropdown.bind(this)}>
-                            <Button color="primary" text>
-                                <i className="fa fa-user" /> {this.getUser()}
+                        <div className="item">
+                        <Button border small onClick={this.navigate.bind(this, '/create/question')}>
+                            <i className="fa fa-pen" /> {t`New question`}
+                        </Button>
+                            {/*<Dropdown
```

Comment:
Remove?

Reply:
I thought about to remove it, but we had a discussion to leave it as it is, when we started the mvp. We had some functionallity at that time in the ui, that was not supported by the backend.